src/mainview.h

```diff
@@ -15,7 +15,7 @@ namespace ui
 	class CMainView : public QDialog
 	{
 		Q_OBJECT;
-
+		QString m_pInstallDir;
```

Comment:
Can we move this down to the private block at the bottom of the class?

Reply:
Fixed